packages/suite-web/package.json

```diff
@@ -6,33 +6,30 @@
         "type-check": "tsc --project tsconfig.json",
         "type-check:watch": "yarn type-check -- --watch",
         "lint": "eslint . --ext .tsx,.ts",
-        "copy-files": "yarn workspace @trezor/suite-data copy-static-files",
-        "dev": "yarn workspace @trezor/suite-data copy-static-files && next",
+        "dev": "rimraf ./build && yarn workspace @trezor/suite-build run dev:web",
+        "dev:analyze": "ANALYZE=true yarn dev",
         "cert": "sh ./scripts/gencert.sh",
-        "dev:https": "yarn workspace @trezor/suite-data copy-static-files && node ./scripts/httpsServer.js",
-        "build:app": "next build && next export -o build",
-        "build": "run-s copy-files build:app"
+        "dev:https": "node ./scripts/httpsServer.js",
+        "build": "rimraf ./build && yarn workspace @trezor/suite-build run build:web"
     },
     "dependencies": {
         "@sentry/browser": "^5.29.2",
         "@sentry/integrations": "^5.29.2",
         "@trezor/suite": "21.5.0",
-        "@zeit/next-workers": "^1.0.0",
-        "next": "^10.0.5",
-        "next-redux-wrapper": "^5.0.0",
-        "styled-components": "5.1.1",
-        "worker-loader": "^3.0.7"
+        "react-helmet": "^6.1.0",
+        "react-redux": "^7.2.2",
+        "react-router": "^5.2.0",
+        "react-router-dom": "^5.2.0",
+        "styled-components": "5.1.1"
     },
     "devDependencies": {
-        "@types/next-redux-wrapper": "^3.0.0",
+        "@trezor/suite-build": "21.5.0",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
+        "@types/react-helmet": "^6.1.0",
         "@types/react-redux": "^7.1.7",
-        "babel-plugin-styled-components": "1.12.0",
-        "git-revision-webpack-plugin": "^3.0.6",
-        "is-website-vulnerable": "^1.14.6",
-        "next-optimized-images": "^2.6.2",
-        "next-transpile-modules": "^4.1.0",
-        "ts-loader": "^8.0.14"
+        "@types/react-router": "^5.1.11",
+        "@types/react-router-dom": "^5.1.7",
+        "@types/ua-parser-js": "^0.7.35"
```

Comment:
How do we get ua-parser-js library? I do not see it in dependencies

Reply:
cf1975a